client/state/index.js

```diff
@@ -109,7 +109,8 @@ const middleware = [ thunkMiddleware, noticesMiddleware ];
 if ( typeof window === 'object' ) {
 	// Browser-specific middlewares
 	middleware.push(
-		require( './analytics/middleware.js' ).analyticsMiddleware
+		require( './analytics/middleware.js' ).analyticsMiddleware,
+		require( './data-layer/wpcom-api-middleware.js' ).middleware,
```

Comment:
the intention was that we would have a `ssr-middleware` which could grab the data directly. could happen synchronously even.

obviously we could keep this in here but if we do it will still let all of those requests through on SSR